tests/poly/lower_poly.mlir

```diff
@@ -4,12 +4,25 @@
 #cycl_2048 = #poly.polynomial<1 + x**1024>
 #ring = #poly.ring<cmod=4294967296, ideal=#cycl_2048>
 module {
-  func.func @test_lower_from_coeffs() {
-    %c0 = arith.constant 0 : index
-    // 2 + 2x + 5x^2
-    %coeffs = arith.constant dense<[2, 2, 5]> : tensor<3xi32>
+  func.func @test_lower_from_coeffs() -> !poly.poly<#ring> {
```

Comment:
could you please add a test case for addition on `tensor<Nxpoly>` ?

Reply:
Updated with a TODO - it's really unfortunate tensors of tensors are not a thing - I filed an issue to discuss what we can do (either detensorize or flatten the tensor)